### PR TITLE
spdk: update linux kernel version requirement

### PR DESCRIPTION
**Pull request by derekbit**
v5.13+ can provide better stability

### CODE REVIEW

File: content/docs/1.5.0/spdk/prerequisites.md [hunk 1]
```diff
@@ -5,11 +5,19 @@ weight: 2
...
 - x86-64 CPU with SSE4.2 instruction support
-  > NOTICE
+  > **NOTICE**
+  >
   > Currently, V2 Data Engine only supports `x86_64` platform.
 
-- Required linux kernel modules
+- Linux kernel
+
+  5.13 or later is required for NVMe over TCP support
```
Comment (innobead) [inline content/docs/1.5.0/spdk/prerequisites.md:17]: Why 5.13+? any critical fixes for that or the following versions?

File: content/docs/1.5.0/spdk/prerequisites.md [hunk 2]
```diff
@@ -5,11 +5,19 @@ weight: 2
 
 ## Prerequisites
 
+Longhorn worker nodes must meet the following requirements:
```
Comment (innobead) [inline content/docs/1.5.0/spdk/prerequisites.md:8]: What does Longhorn worker node mean? Do you know nodes running with instance manager?

Comment (derekbit) [reply content/docs/1.5.0/spdk/prerequisites.md:8]: Yes.

File: content/docs/1.5.0/spdk/prerequisites.md (hunk 1 above)
Comment (derekbit) [reply content/docs/1.5.0/spdk/prerequisites.md:17]: there are many bug fixes in older versions and mayastor also encounter issues in old kernel
We can use 5.13 for safety.

https://github.com/openebs/mayastor/releases

**Review by innobead — APPROVED** [review]

**Pull request merged**